### ISSUE TITLE
Valid JSON SPDX file cannot be converted to tag:value

### ISSUE THREAD

**Issue by vargenau** [spdx/spdx-online-tools#385]
About:
```
Spdx online tools version : 1.0.7
Java Tools version : 1.1.0
```

Attached JSON file is valid according to the online validator.

When trying to convert to tag:value, I get the following error:

```
Analysis exception processing SPDX file: Value must start with one of NOASSERTION, Person:, Organization:
```

[busybox-minimal-apko-sbom-x86_64.spdx.json.txt](https://github.com/spdx/spdx-online-tools/files/9382496/busybox-minimal-apko-sbom-x86_64.spdx.json.txt)

**Comment by BanulaKumarage** [comment]
@goneall this issue still raises when I tried. Did you find any findings regarding this?

**Comment by BanulaKumarage** [comment]
This issue raises in the latest branch of SPDX online tools. But when I converted this using the Java 1.1.0 release, it was converted. 

![image](https://user-images.githubusercontent.com/63106638/230788916-1f772e86-176d-4855-b9b3-6ac57e12ebc3.png)


@goneall Can you guide me on working on this issue?

**Comment by goneall** [comment]
The error is actually in validate.  The document isn't valid, but it showed as valid in the online tools.  This is fixed in the java tools in https://github.com/spdx/Spdx-Java-Library/pull/159.  Here's the output running the code from the latest (unreleased) tools-java:

```
This SPDX Document is not valid due to:
	Relationship error: Relationship error: Originator error - Value must start with one of NOASSERTION, Person:, Organization: for package busybox in sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a in sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a in sbom-sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a
	Relationship error: Relationship error: Originator error - Value must start with one of NOASSERTION, Person:, Organization: for package musl in sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a in sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a in sbom-sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a
	Relationship error: Relationship error: Originator error - Value must start with one of NOASSERTION, Person:, Organization: for package alpine-baselayout-data in sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a in sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a in sbom-sha256:6552f62b46d2f1edc7cc16d591e325e960e87777aed3b2f37c72cfe4c328763a
```

I'll need to spin up a new release of the java tools and update this repo once it is release.  I see if I can get that done in the next couple of days.

**Comment by BanulaKumarage** [comment]
@goneall does that mean, we can close this issue with the release?

**Comment by goneall** [comment]
Yes - once it is release.

**Comment by goneall** [comment]
fixed with PR #517
